src/table/selections-utils.js

```diff
@@ -38,37 +38,51 @@ export function getSelectionClass(cell, selState) {
 export function reducer(state, action) {
   switch (action.type) {
     case 'select':
-      return { ...state, rows: action.payload.newRows, colIdx: action.payload.colIdx };
+      return { ...state, rows: action.payload.selectedRows, colIdx: action.payload.colIdx };
     case 'reset':
       return state.rows.length ? { ...state, rows: [], colIdx: -1 } : state;
     default:
       throw new Error('reducer called with invalid action type');
   }
 }
 
-export function selectCell(cell, selState, selDispatch) {
+export const getSelectedRows = (selectedRows, qElemNumber, rowIdx, rows, evt) => {
+  const alreadySelectedIdx = rows.findIndex((r) => r.qElemNumber === qElemNumber);
```

Comment:
This is actually a typo, it should be `selectedRows.findIndex`. So you don't have to sent `rows` as an argument

Reply:
will fix that. Thanks.